client/clientservice/src/event_service.cpp

```diff
@@ -40,6 +40,7 @@ namespace concord::client::clientservice {
 Status EventServiceImpl::Subscribe(ServerContext* context,
                                    const SubscribeRequest* proto_request,
                                    ServerWriter<SubscribeResponse>* stream) {
+  LOG_INFO(logger_, "");
```

Comment:
Should anything from input params be logged here?

Reply:
context is pretty opaque, used to call APIs (RPC type).
The response is not yet filled.
proto_request holds a variant, so maybe can add info later but not in this line, it depends what mode we work with.
I'll see if I can add some logs later on the if/else.
Thanks.